### DISCUSSION TITLE
进来了！我进来了！哈哈哈哈哈哈

### DISCUSSION THREAD

**Discussion by xiaomei-199** [share121/inter-knot#828] [其他]
![1723281862735](https://github.com/user-attachments/assets/1466a034-0c7b-45a7-bf1d-feddbfb2e794)

**Comment by RS64671012** [comment]
收起你的龙图（

**Comment by tianmu666** [comment]
不知道为什么绳网的入站手续这么复杂

**Reply by Takarada-Rikka** [reply]
因为这和一般的网页不一样，你也不看看是用什么登录的

**Comment by dongzhiaidao** [comment]
复杂点好，起码挡了一部分找麻烦的人

**Comment by infinite-vector** [comment]
毕竟是见不得人的绳网嘛，肯定不能让大众都能轻易进来呗！

**Comment by shabo1919** [comment]
绳网太容易进来就得打帝皇战争了

**Comment by Elysia-Ego-093** [comment]
复杂就好啊，省得那些不明生物带节奏
